Ark.Tools.AspNetCore/Startup/Ex.cs

```diff
@@ -180,5 +180,46 @@ public static IServiceCollection ArkConfigureSwaggerAuth0(this IServiceCollectio
 
             return services;
         }
+
+        public static IServiceCollection ArkConfigureSwaggerAzureB2C(this IServiceCollection services, string instance, string domain, string clientId, string signUpSignIn, string apiId)
+        {
+            services.ConfigureSwaggerGen(c =>
+            {
+                var oauthScheme = new OpenApiSecurityScheme
+                {
+                    Type = SecuritySchemeType.OAuth2,
+                    
+                    Flows = new OpenApiOAuthFlows()
+                    {
+                        AuthorizationCode = new OpenApiOAuthFlow()
+                        {
+                            AuthorizationUrl = new Uri($"{instance}/{domain}/{signUpSignIn}/oauth2/v2.0/authorize"),
+                            TokenUrl = new Uri($"{instance}/{domain}/{signUpSignIn}/oauth2/v2.0/token"),                            
+                            Scopes = new Dictionary<string, string>
+                            {
+                                { "openid", "Grant access to user" },
+                                { $"https://{domain}/{apiId}/default", "Default scope to retrieve user permissions" }

```

Comment:
Thinking about it, I don't like 'default' as it doesn't convey anything.
Azure old suggestion was 'user_impersonate' which isn't grand either.

How about 'access_as_user' ?